content/docs/run-api-server/Ingestion-filtering.mdx

```diff
@@ -0,0 +1,147 @@
+
+To add new filtered entities, submit an `HTTP PUT` request to the admin API endpoints for either Asset or Account filters. The PUT request body will be JSON that expresses the filter rules, currently the rules model is a whitelist format and expressed as JSON string array. 
+To remove entities, submit an `HTTP PUT` request to update the list accordingly. 
+To retrieve what is currently configured, submit an `HTTP GET` request.
+  
+The OAPI doc published by the Admin Server can be pulled directly from the Github repo here:
+https://github.com/stellar/go/blob/horizon-v2.18.0/services/horizon/internal/httpx/static/admin_oapi.yml
+
+### Reverting Options:
+
+1. Disable both Asset and Account Filter config rules via the [Admin API](https://github.com/stellar/go/blob/master/services/horizon/internal/httpx/static/admin_oapi.yml) by setting `enabled=false` in each filter rule, or set `--exp-enable_ingestion_filtering=false`, this will open up forward ingestion to include all data again. It is then your choice whether to run a Re-ingestion to capture older data from past that would have been dropped by filters but could now be re-imported with filters off, e.g. `horizon db reingest <from_ledger> <to_ledger>`
+
+2. If you have a DB backup:
+ - restore the DB 
+ - run a Reingestion Gap Fill command to fill in the gaps to current tip of the chain
+ - resume Ingestion Sync
+
+3. Start over with a fresh DB (or see Patching Historical Data below)
+  
+### Patching Historical Data:
+If new Assets or Accounts are added to the whitelist and you would like to patch in its missing historical data, Reingestion can be run. The Reingestion process is idempotent and will re-ingest the data from the designated ledger range and overwrite or insert new data if not already on current DB.
+ 
+## Sample Use Case:
+As an Asset Issuer, I have issued 4 assets and am interested in all transaction data related to those assets including customer Accounts that interact with those assets and the following:
+ - Operations
+ - Effects
+ - Payments
+ - Claimable balances
+ - Trades
+
+I would like to store the full history of all transactions related from the genesis of those assets. 
+
+### Pre-requisites:
+You have an existing Horizon installed, configured and has forward ingestion enabled at a minimum to be able to successfully sync to the current state of the Stellar network. Bonus if you are familiar with running re-ingestion. 
+
+Steps:
+  
+1. Configure 4 whitelisted Assets via the Admin API. Also check the `HISTORY_RETENTION_COUNT` and set it to `0` if you don’t want any history purged anymore now that you are filtering, otherwise it will continue to reap all data older than the retention.
+
+2. Decide if you want to wipe existing history data on the DB first before the filtering starts running, you can effectively clear the history by running 
+
+  <CodeExample>
+  
+ ```   
+  ‘history-retention-count=1 stellar-horizon db reap’ 
+ ```
+  
+  </CodeExample>
```

Comment:
It could be the space prefix here. Try that and I'll check Jenkins in the meantime